### ISSUE TITLE
Add Grafana dashboard for Blackbox exporter probes

### ISSUE THREAD

**Issue by nanorobocop** [prometheus-operator/kube-prometheus#1575]
**What is missing?**
Grafana dashboard for Blackbox exporter probes

**Why do we need it?**
To monitor probes info

**Environment**
Any

* kube-prometheus version:
any, main branch

**Anything else we need to know?**:

**Comment by paulfantom** [comment]
Dashboards can be added individually.

Since blackbox_exporter does not ship any monitoring mixin, we don't include any dashboard either. If you have a good, generic dashboard in mind, please share.

**Comment by nanorobocop** [comment]
There is public one, which is not bad - https://grafana.com/grafana/dashboards/7587.
When any probes added - it could display blackbox exporter metrics out of the box.

**Comment by paulfantom** [comment]
I agree that one is a quite good dashboard for blackbox_exporter. Do you know where is the git repository for the dashboard? We need it to check the license and include json file programmatically.

**Comment by nanorobocop** [comment]
I tried to search original repo, but no luck.
Also not found exact info about license for community dashboards.
